src/AppLayout.tsx

```diff
@@ -16,40 +19,147 @@ import {
   BreadcrumbItem,*/
   PageSidebar,
 } from '@patternfly/react-core';
-import { CogIcon, OutlinedQuestionCircleIcon, BellIcon } from '@patternfly/react-icons';
+import { BellIcon, CaretDownIcon, CogIcon, OutlinedQuestionCircleIcon, UserIcon } from '@patternfly/react-icons';
 import { Link, useLocation } from 'react-router-dom';
 import pncLogoText from './pnc-logo-text.svg';
+import { AboutModalPage } from './components/AboutModalPage/AboutModalPage';
+
+// extend the global Window interface
+declare global {
+  interface Window {
+    pnc?: any;
+  }
+}
 
 interface IAppLayout {
   children: React.ReactNode;
 }
 
 export const AppLayout: React.FunctionComponent<IAppLayout> = ({ children }) => {
+  const pncWebConfigLoadedFromOrch = window.pnc;
+
   const AppLogoImage = () => <img src={pncLogoText} alt="Newcastle Build System" />;
 
-  const AppHeaderTools = (
-    <PageHeaderTools>
-      <PageHeaderToolsGroup>
-        <PageHeaderToolsItem>
-          <Button variant={ButtonVariant.plain}>
-            <CogIcon />
-          </Button>
-        </PageHeaderToolsItem>
-        <PageHeaderToolsItem>
-          <Button variant={ButtonVariant.plain}>
-            <OutlinedQuestionCircleIcon />
-          </Button>
-        </PageHeaderToolsItem>
-        <PageHeaderToolsItem>
-          <Button variant={ButtonVariant.plain}>
-            <BellIcon />
-          </Button>
-        </PageHeaderToolsItem>
-      </PageHeaderToolsGroup>
-    </PageHeaderTools>
-  );
+  const AppHeaderTools = () => {
+    const pncUserGuideUrl = pncWebConfigLoadedFromOrch.config.userGuideUrl;
+
+    const [isHeaderConfigOpen, setIsHeaderConfigOpen] = useState(false);
+    const [isHeaderQuestionOpen, setIsHeaderQuestionOpen] = useState(false);
+    const [isHeaderUserOpen, setIsHeaderUserOpen] = useState(false);
+    const [isAboutModalOpen, setIsAboutModalOpen] = useState(false);
+    const [currentUser, setCurrentUser] = useState<any>(null);
+
+    const headerConfigDropdownItems = [
+      <DropdownItem key="variables" href="/admin/variables">
```

Comment:
please use component coming from react-router library to make sure base name is applied